sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/ColumnPruningSuite.scala

```diff
@@ -35,7 +35,8 @@ class ColumnPruningSuite extends PlanTest {
   object Optimize extends RuleExecutor[LogicalPlan] {
     val batches = Batch("Column pruning", FixedPoint(100),
       ColumnPruning,
-      CollapseProject) :: Nil
+      CollapseProject,
+      PushPredicateThroughProject) :: Nil
```

Comment:
I will put them in the same order than Optimizer to make sure that i does not hide any potential issue.